apiserver/params/params.go

```diff
@@ -1383,3 +1383,24 @@ type UpgradeSeriesUnitsResult struct {
 	Error     *Error   `json:"error,omitempty"`
 	UnitNames []string `json:"unit-names"`
 }
+
+type ProfileChangeResult struct {
+	AppName        string          `json:"app-name"`
+	OldProfileName string          `json:"old-profile-name"`
+	CharmRev       int             `json:"charm-rev"`
+	Profile        CharmLXDProfile `json:"profile"`
```

Comment:
should this and `OldProfileName` be omitempty, as we're not guaranteed to have them.